tracee-rules/main.go

```diff
@@ -168,6 +168,10 @@ func main() {
 				Name:  "rego-enable-parsed-events",
 				Usage: "enables pre parsing of input events to rego prior to evaluation",
 			},
+			&cli.BoolFlag{
+				Name:  "rego-aio-enable",
```

Comment:
NIT: Maybe we can keep this flag shorter without loosing context by replacing `--rego-aio-enable` with `--rego-aio`. I was also thinking if `--rego-parse-events` can be used instead of `--rego-enable-parsed-events`. WDYT?

Reply:
Sure, updated.